<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LexicalEntrySearchDialog</class>
 <widget class="QDialog" name="LexicalEntrySearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search for a lexical entry</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="WritingSystemCombo" name="writingSystemCombo"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="LingEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
   <item>
    <widget class="QListView" name="lowerListView"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LingEdit</class>
   <extends>QLineEdit</extends>
   <header>lingedit.h</header>
  </customwidget>
  <customwidget>
   <class>WritingSystemCombo</class>
   <extends>QComboBox</extends>
   <header>writingsystemcombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>lowerListView</tabstop>
  <tabstop>writingSystemCombo</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>listView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LexicalEntrySearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LexicalEntrySearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
